### PR TITLE
Default to scr.color=COLOR_MODE_16 for Alacritty on Windows

### PR DESCRIPTION
**Pull request by kazarmy**
Reason given in code comments.

### CODE REVIEW

**Comment by radare** [comment]
Scr.color should be int

**Comment by kazarmy** [comment]
ok 6d57a67bb0a5337e92a914b2bbd2a1cecf87c97e

File: libr/main/radare2.c [hunk 1]
```diff
@@ -399,6 +399,17 @@ static bool mustSaveHistory(RConfig *c) {
 
 // Try to set the correct scr.color for the current terminal.
 static void set_color_default(void) {
+#ifdef __WINDOWS__
+	char *alacritty = r_sys_getenv ("ALACRITTY_LOG");
```
Comment (radare) [inline libr/main/radare2.c]: thats leaking, you should free that alacritty thing

Comment (kazarmy) [reply libr/main/radare2.c]: sorry didn't realize this

**Pull request merged**